<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SIMULATION</class>
 <widget class="QMainWindow" name="SIMULATION">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SIMULATION v0.6.4</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" colspan="5">
     <widget class="QTableWidget" name="Display_Table">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>49</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ESPECIE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>INDIVIDUOS TOTALES</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QPushButton" name="NEXT_Button">
      <property name="text">
       <string>SIGUIENTE PASO</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLCDNumber" name="GEN"/>
    </item>
    <item row="2" column="4">
     <widget class="QSpinBox" name="GEN_STEP">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>GENERACIONES POR PASO</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="MUT_Button">
      <property name="text">
       <string>MUTACIÓN</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="5">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="4" colspan="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="Actual">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="Graph_Check">
      <property name="text">
       <string>REPRESENTACIÓN GRÁFICA</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>992</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
